### ISSUE TITLE
Line plot scaling issue when WebGL is used 

### ISSUE THREAD

**Issue by papahabla** [bokeh/bokeh#4248]
When enabling WebGL in the following code, the y axis labels are correct, but the scaling of the line plot y data is incorrect. If `webgl=False` or `scatter` is used in place of `line`, the data is presented correctly. 

```python
import numpy as np
from bokeh.plotting import figure, show
from bokeh.io import output_notebook
from bokeh.io import push_notebook

output_notebook()

fs = 48000
t = .1
x = np.random.standard_normal(size=(1, int(np.round(t*fs)))) * 0.25
print(x)
print('MAX ABS: {}'.format(np.max(np.abs(x))))

p = figure(title='Time Signal', plot_height=300, plot_width=800, webgl=True)
r = p.line(np.linspace(0, t, x.shape[1]), x[0, :])
show(p)
```

OS: Mac 10.10.5
Browser: Chrome 50.0.2661.86 (64-bit)
Bokeh: 0.11.1

**Comment by bryevdv** [comment]
ping @almarklein

**Comment by almarklein** [comment]
@papahabla can you please explain "incorrect", maybe via a screenshot?

**Comment by papahabla** [comment]
@almarklein 

Below are images of the above described OS, Browser, Bokeh. 

**Chrome - webgl=False**
![test_bokeh_gl_false](https://cloud.githubusercontent.com/assets/10580407/14992332/80f7d92c-111a-11e6-893c-3711de16fc59.png)


**Chrome - webgl=True**
![test_bokeh_gl_true](https://cloud.githubusercontent.com/assets/10580407/14992385/b903d654-111a-11e6-842e-7b74df70df84.png)


Also, when using Safari with `webgl=True', the plot lines do not draw.

**Safari 9.1 - webgl=True**

![test_bokeh_safari_gl_true](https://cloud.githubusercontent.com/assets/10580407/14992574/79baae0e-111b-11e6-9446-874d766103a7.png)

**Comment by almarklein** [comment]
Ok, this is a known issue in 0.11; sharp corners result in long miters which are not truncated. This was fixed in #3888 (will be in 0.12). WebGL is know not to work (well) in Safari, unfortunately.

**Comment by almarklein** [comment]
Ok, this is a known issue in 0.11; sharp corners result in long miters which are not truncated. This was fixed in #3888 (will be in 0.12). WebGL is know not to work (well) in Safari, unfortunately.

**Comment by almarklein** [comment]
@papahabla Thanks for the report though!
